src/Vault.sol

```diff
@@ -293,9 +294,10 @@ contract Vault {
     function getCollateralValue() public view returns (uint256 collateralValue) {
```

Comment:
To be discussed if getCollateralValue() should be dynamic or locked in at the moment the moment the position is opened.